akka-stream/src/main/scala/akka/stream/javadsl/SubFlow.scala

```diff
@@ -561,7 +561,7 @@ class SubFlow[-In, +Out, +Mat](delegate: scaladsl.SubFlow[Out, Mat, scaladsl.Flo
 
   /**
    * Terminate processing (and cancel the upstream publisher) after predicate
-   * returns false for the first time. Due to input buffering some elements may have been
+   * returns false for the first time (or 1 after if inclusive). Due to input buffering some elements may have been
```

Comment:
Suggestion:
`* returns false for the first time, including the first failed element iff inclusive is true.`